lib/common/exceptions/internal/evss_forbidden.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module Common
```

Comment:
should probably live in the lib/evss dir (with that namespacing) as it's not common, or if it is common that we want a 'visible' forbidden exception maybe rename it.